src/Hl7.Fhir.Core/Serialization/ComplexTypeWriter.cs

```diff
@@ -87,15 +81,20 @@ private void write(ClassMapping mapping, object instance, bool summary, Property
 
                 // Now, if our writer does not use dual properties for primitive values + rest (xml),
                 // or this is a complex property without value element, serialize data normally
-                if(!_writer.HasValueElementSupport || !serializedIntoTwoProperties(prop,value))
+                if (!_writer.HasValueElementSupport || !serializedIntoTwoProperties(prop, value))
+                {
                     writer.Serialize(prop, value, summary, SerializationMode.AllMembers);
+                }
                 else
                 {
                     // else split up between two properties, name and _name
-                    writer.Serialize(prop,value, summary, SerializationMode.ValueElement);
+                    writer.Serialize(prop, value, summary, SerializationMode.ValueElement);
                     _writer.WriteEndProperty();
                     _writer.WriteStartProperty("_" + memberName);
-                    writer.Serialize(prop, value, summary, SerializationMode.NonValueElements);
+                    if (!isFhirPrimitive(value))
+                    {
+                        writer.Serialize(prop, value, summary, SerializationMode.NonValueElements);
```

Comment:
An example use could be to indicate a reason for an Organization being inactive
 <Organization>
      <active value=false>
             <extension url="http://example.org/inactivereason">
                      <valueString value="Organization was de-registered by AHPRA"/>
             </extension>
      </active>